ansible/playbooks/setup-node.yaml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Install Node and other dependencies
+  hosts: 127.0.0.1
+  connection: local
+  become: true
+
+  vars:
+    node_version: "16.*"
```

Comment:
I think there was some very subtle behavior difference people were getting running unit tests between minor versions of node. Is it possible in ansible to specify a more specific version or do we have to just live with 16.* (same question for pnpm version but I don't think we've had any issue there)

Reply:
+1 if we can specify a specific version, I think we win to rule out small changes in behavior across versions – since our systems are offline, the incremental win we get from automatic minor version updates are not as high as the risk of behavior changes.

Reply:
@benadida, @carolinemodic, totally agree that pinning versions is ideal. It does not appear to be the way Node expects installation to go though: https://github.com/nodesource/distributions#using-debian-as-root-2

We can certainly pin to a major.minor.patch version, but we currently do not do that in `vxsuite-complete-system`, even though we do in `vxsuite`.  I have pushed up an example pinning it to `16.19.0` (which is what a new build in `vxsuite-complete-system` would use), but since we do a very manual install in `vxsuite` to `16.4.2`, I'd like to get more feedback and tests.